index.html

```diff
@@ -170,7 +164,7 @@
                   id="q20"
                   data-name="JavaScript"
                   class="q20"
-                ><div key="0">JavaScript</div>
+                ><div>JavaScript</div>
                 </div>
                 </div>
               </div>
```

Comment:
- [ ] [OPTIONAL] Project technologies should be in a `<ul>` tags. Please consider making this change for every project listed.

Example code:
```HTML
    <ul>
      <li>Tech one</>
      <li>Tech two</>
      <li>Tech three</>
    </ul>
```